.github/workflows/build.yml

```
@@ -85,4 +94,10 @@
         uses: actions/upload-artifact@v4
         with:
           name: pr_id
-          path: pr/
+          path: pr/
+          retention-days: 1
+
+      - name: Clean up cache
+        run: |
+          rm -rf ./.buildx-cache
+          mv ./.buildx-cache-new /tmp/.buildx-cache
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)